index.html

```diff
@@ -99,9 +99,9 @@ <h1 class="greeting">
             <section class="work-section">
```

Comment:
- [ ] In this section again, we are still using the resizing percentage of at least 150%. I have noticed that the `info li` text overlaps with the `owner-name`. In addition, I can see that the ruby on rails text is crossing beyond its borders due to the fixed height of 24px given to it. I suggest ensuring that the texts are well structured to avoid overlapping. This will increase the portfolio accessibility and be user-friendly.
- [ ] Finally ensure the same text overlap is fixed on the `about section` text referring to the attached screenshot of about section.
Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/231137411-2cc3a0f0-8723-47ff-941d-aa866e2f4e19.png)

About section
![image](https://user-images.githubusercontent.com/99394374/231137705-0c3e0e26-ea1e-4dac-8ffe-f49232846d4b.png)